client/src/views/users/UserListDisplay.vue

```diff
@@ -69,9 +69,14 @@ const hotkeyManager: HotkeyManager = inject(HotkeyManagerKey)!;
 let hotkeys: Hotkeys | null = null;
 
 const selectedCount = ref(0);
+const activeUsersCount = props.users.getUsers().filter((user) => !user.isRevoked()).length;
 
 const allSelected = computed(() => {
-  return selectedCount.value === props.users.usersCount();
+  if (selectedCount.value === activeUsersCount && selectedCount.value > 0) {
```

Comment:
You can just `return selectedCount.value > 0 && selectedCount.value === activeUsersCount`